test/python/WMCore_t/WMBS_t/Subscription_t.py

```diff
@@ -1065,29 +1065,20 @@ def testListIncompleteDAO(self):
 
         incompleteSubs = subIncomplete.execute()
 
-        assert len(incompleteSubs) == 2, \
-            "ERROR: Wrong number of incomplete subscriptions returned: %s" % len(incompleteSubs)
-        assert testSubscription["id"] in incompleteSubs, \
-            "ERROR: Original subscription is missing."
-
+        self.assertEqual(len(incompleteSubs), 2, "ERROR: Wrong number of incomplete subscriptions returned: %s" % len(incompleteSubs))
```

Comment:
I never see the point in putting "ERROR:" at the front of these. The assertion error kind of shows there is an error....

Reply:
Agreed! Will fix it